debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/RomClassWalker.java

```diff
@@ -702,7 +702,12 @@ void allSlotsInOptionalInfoDo() throws CorruptDataException
 			permittedSubclassAttributeDo(U32Pointer.cast(cursor.get()));
 			cursor = cursor.add(1);
 		}
-
+/*[IF INLINE-TYPES]*/
+		if (romClass.optionalFlags().allBitsIn(J9NonbuilderConstants.J9_ROMCLASS_OPTINFO_INJECTED_INTERFACE_INFO)) {
```

Comment:
look at `openj9/debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/pointer/helper/ValueTypeHelper.java` you can use `areValueTypesSupported` to determine if VT is enabled for the core file